brightcove-exoplayer/BasicCastSampleApp/build.gradle

```diff
@@ -1,9 +1,18 @@
+android {
+    defaultConfig {
+        minSdkVersion ANDROID_MIN_SDK_VERSION
+        targetSdkVersion ANDROID_TARGET_SDK_VERSION
+        multiDexEnabled true
+    }
+}
+
 dependencies {
     implementation "com.brightcove.player:android-appcompat-plugin:${anpVersion}"
     implementation "androidx.appcompat:appcompat:${ANDROIDX_APPCOMPAT}"
     implementation "androidx.legacy:legacy-support-v4:${ANDROIDX_LEGACY_SUPPORT_V4}"
     implementation "androidx.recyclerview:recyclerview:${ANDROIDX_RECYCLER_VIEW}"
     implementation "androidx.vectordrawable:vectordrawable:${ANDROIDX_VECTOR_DRAWABLE}"
     implementation "androidx.mediarouter:mediarouter:${ANDROIDX_MEDIA_ROUTER}"
-    implementation 'com.squareup.picasso:picasso:2.5.2'
+    implementation 'com.squareup.picasso:picasso:2.71828'
```

Comment:
@BCjwhisenant Can you create a variable for the Picasso and multidex libraries, it will be helpful for maintenance.

Reply:
done, thanks for that suggestion